### PR TITLE
OCPBUGS-4520: Update jenkins master branch to use 4.13 base image

### PR DESCRIPTION
**Pull request by jkhelil**

### CODE REVIEW

**Review by otaviof — APPROVED** [review]

File: ci-operator/step-registry/jenkins/sync-plugin/e2e/jenkins-sync-plugin-e2e-ref.yaml [hunk 1]
```diff
@@ -10,5 +10,11 @@ ref:
   dependencies:
   - name: "pipeline:jenkins"
     env: "JENKINS_IMAGE"
+  - name: jenkins-agent-base
+    env: JENKINS_AGENT_BASE_IMAGE
+  - name: jenkins-agent-maven
+    env: JENKINS_AGENT_MAVEN_IMAGE
```
[the remark this answers is not in the document]
Comment (jkhelil) [reply ci-operator/step-registry/jenkins/sync-plugin/e2e/jenkins-sync-plugin-e2e-ref.yaml]: done

**Comment by gabemontero** [comment]
ok sync plugin e2e passed per https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/34493/rehearse-34493-pull-ci-openshift-jenkins-master-e2e-aws-jenkins-sync-plugin/1600138316676599808 @jkhelil so good news there

that said, I just realized you need to add the agent env var to https://github.com/openshift/release/blob/master/ci-operator/step-registry/jenkins/client-plugin/tests/jenkins-client-plugin-tests-ref.yaml for the client plugin as well just in case / for the future, though the test passed per https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/34493/rehearse-34493-pull-ci-openshift-jenkins-master-e2e-aws-jenkins-client-plugin/1600138316546576384/artifacts/e2e-aws-jenkins-client-plugin/jenkins-client-plugin-tests/build-log.txt and the test failed after the fact in https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/34493/rehearse-34493-pull-ci-openshift-jenkins-master-e2e-aws-jenkins-client-plugin/1600138316546576384 in some post processing.

I see the `No jenkins agent base image env var found, not overriding jenkins-agent-base imagestream for e2e test of jenkins-client-plugin.` in the https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/34493/rehearse-34493-pull-ci-openshift-jenkins-master-e2e-aws-jenkins-client-plugin/1600138316546576384/artifacts/e2e-aws-jenkins-client-plugin/jenkins-client-plugin-tests/build-log.txt

**Comment by gabemontero** [comment]
/lgtm

the e2s-aws failure is unrelated (ec2 failure on cluster install); all the jenkins e2e's passed.

**Comment by otaviof** [comment]
> /lgtm
> 
> the e2s-aws failure is unrelated (ec2 failure on cluster install); all the jenkins e2e's passed.

/override ci/rehearse/openshift/jenkins/master/e2e-aws

**Comment by gabemontero** [comment]
the tests have passed when cluster install did not flake

**Pull request merged**